contrib/platform/test/com/sun/jna/platform/win32/WevtapiTest.java

```diff
@@ -0,0 +1,574 @@
+package com.sun.jna.platform.win32;
+
+import com.sun.jna.Memory;
+import com.sun.jna.platform.win32.WinDef.BOOL;
+import com.sun.jna.platform.win32.Winevt.EVT_CHANNEL_CONFIG_PROPERTY_ID;
+import com.sun.jna.platform.win32.Winevt.EVT_HANDLE;
+import com.sun.jna.ptr.IntByReference;
+import junit.framework.TestCase;
+
+import java.io.File;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+import static org.hamcrest.CoreMatchers.nullValue;
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.assertThat;
+
+/**
+ * @author Minoru Sakamoto
+ */
+public class WevtapiTest extends TestCase {
+
+    public void testEvtGetExtendedStatus() throws Exception {
+        assertThat(_evtGetExtendedStatus(null).length(), is(0));
+        assertThat(_evtGetExtendedStatus(">><").length() > 0, is(true)); // illegal query
+    }
+
+    private String _evtGetExtendedStatus(String query) {
+        EVT_HANDLE handle = null;
+        String result;
+        try {
+            handle = Wevtapi.INSTANCE.EvtQuery(null, "Application", query,
+                    Winevt.EVT_QUERY_FLAGS.EvtQueryChannelPath);
+            result = WevtapiUtil.EvtGetExtendedStatus();
+        } finally {
+            if (handle != null) {
+                Wevtapi.INSTANCE.EvtClose(handle);
+            }
+        }
+        System.out.println(result);

```

Comment:
This looks like an artifact from debugging. I was surprised when I saw the output and could not directly place it. Please remove.